drivers/drive-lorenz.c

```diff
@@ -362,7 +360,7 @@ int main (int argc, char *argv[])
    double        tstart, tstop;
    int           ntime;
 
-   int           max_levels = 1;
+   int           max_levels = 2;
```

Comment:
Just curious...is this driver file redundant now?  No one really uses it, so it could get eliminated, assuming it's funcationality is now duplicated by example 7 (and the other Lorenz driver in this directory)